modules/vaos/app/services/vaos/v2/systems_service.rb

```diff
@@ -14,11 +14,11 @@ def get_facility_clinics(location_id:,
         end
       end
 
-      def get_available_slots(location_id:, clinic_id:, clinical_service:, start_dt:, end_dt:)
+      def get_available_slots(location_id:, clinic_id:, clinical_service:, provider_id:, start_dt:, end_dt:)
```

Comment:
We probably need to add an exception to this and `get_slots_vpg` to bypass the linter.

Reply:
You can also refactor `get_available_slots` to accept a single options hash instead of multiple keyword arguments:
```
def get_available_slots(options = {})
  location_id      = options[:location_id]
  clinic_id        = options[:clinic_id]
  clinical_service = options[:clinical_service]
  provider_id      = options[:provider_id]
  start_dt         = options[:start_dt]
  end_dt           = options[:end_dt]
```